src/main/java/uk/gov/hmcts/ccd/domain/service/common/AccessControlServiceImpl.java

```diff
@@ -86,7 +86,7 @@ public boolean canAccessCaseStateWithCriteria(final String caseState,
         boolean hasAccess = hasAccessControlList(accessProfiles, criteria, stateACLs);
 
         if (!hasAccess) {
-            LOG.error(NO_ROLE_FOUND, "caseState",
+            LOG.debug(NO_ROLE_FOUND, "caseState",
                     caseState,
                     AccessControlService.extractAccessProfileNames(accessProfiles),
                     "caseStateACL",
```

Comment:
General note for other similar logs to the ones changed in this class:
- `CaseEvent` filtering logs only ~300 times in the last 7 days.
- `CaseField` filtering logs only ~20 times in the last 7 days. 

Both of these are messages that might be useful for debugging scenarios, and given the very low occurrence compared to the millions of others, suggest we keep them.